src/ari_sxn_simbeamline/xrt_sim/bl_initialization.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import xrt.backends.raycing.materials as xrt_material
+
+
+nickel = xrt_material.Material('Ni', rho=8.908, table='Chantler Total', kind='mirror', name='Ni')
+genericGR = xrt_material.Material('Ni', rho=8.908, table='Chantler total', kind='grating',
+                        name='generic grating', efficiency=[(1, 1), (-1, 1)])  # efficiency = 1
+
+def BLparams():
+    '''Initialize the beamline parameters!
+
+    1. blG is a dictionary containing all parameters for beamline simulation.
+
+    2. update_para is a dictionary containing the parameters that are modified
+    comparing to their initial values. update_para should be a sub-dict of blG.
+
+    Note: The distance is in the unit of mm,
+        and the angle is in the unit of radian.
+    '''
+    blG = dict()
+    ############################################################################
```

Comment:
Again I would prefer not to include these comment lines here and below

Reply:
@awalter-bnl All raised issues have been resolved, it should be ready to be merged.